tcf_core/settings/__init__.py

```diff
@@ -17,7 +17,9 @@ def custom_recording_logic(request):
     return not request.path.startswith('/api')
 
 
-# Performance profiling for non-API views during development
-INSTALLED_APPS.append('silk')
-MIDDLEWARE.append('silk.middleware.SilkyMiddleware')
-SILKY_INTERCEPT_FUNC = custom_recording_logic
+if os.environ.get('DJANGO_SETTINGS_MODULE') not in [
+        'tcf_core.settings.dev', 'tcf_core.settings.prod']:
+    # Performance profiling for non-API views during development
```

Comment:
I think this comment should be above the `if` line because it describes the functionality of the entire block.